lib/mocharc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "diff": true,
+  "extension": ["js"],
+  "opts": "./test/mocha.opts",
+  "package": "./package.json",
+  "reporter": "spec",
+  "slow": 75,
+  "timeout": 2000,
+  "ui": "bdd"
+}
```

Comment:
Just noticed this file. Why was it placed in "/lib" rather than in "/config" (since it's not source code)?

Reply:
because it didn’t seem like it needed a new directory containing a single file